services-custom/s3-transfermanager/src/it/java/software/amazon/awssdk/custom/s3/transfer/TransferManagerTestUtils.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.custom.s3.transfer;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.security.MessageDigest;
+import software.amazon.awssdk.utils.BinaryUtils;
+
+public final class TransferManagerTestUtils {
```

Comment:
Adding javadoc to this file might be a bit unnecessary because the class name seems descriptive enough. I'll add javadocs for other classes if the name doesn't reveal the intention.